dogfood/main.tf

```diff
@@ -95,48 +95,97 @@ resource "coder_agent" "dev" {
 
   metadata {
     display_name = "CPU Usage"
-    key          = "cpu"
-    script       = <<EOT
-    vmstat | awk 'FNR==3 {printf "%2.0f%%", $13+$14+$16}'
-    EOT
-    interval     = 1
+    interval     = 10
     timeout      = 1
+    key          = "0_cpu_usage"
+    script       = <<EOT
+      #!/bin/bash
+      # interval in microseconds should be metadata.interval * 1000000
+      interval=10000000
+      ncores=$(nproc)
+      cusage_p=$(cat /tmp/cusage || echo 0)
```

Comment:
If we haven't calculated yet, let's just output "Unknown", write to /tmp/cusage, and exit early.